src/System.Net.NetworkInformation/src/System.Net.NetworkInformation.csproj

```diff
@@ -228,21 +228,113 @@
+    <Compile Include="System\Net\NetworkInformation\LinuxIPInterfaceStatistics.cs" />
+    <Compile Include="System\Net\NetworkInformation\NetworkInterfacePal.Linux.cs" />
+    <Compile Include="System\Net\NetworkInformation\LinuxNetworkInterface.cs" />
+    <Compile Include="System\Net\NetworkInformation\LinuxIPv4InterfaceProperties.cs" />
+    <Compile Include="System\Net\NetworkInformation\LinuxIPv6InterfaceProperties.cs" />
+    <!-- Linux Common -->
+    <Compile Include="$(CommonPath)\System\IO\StringParser.cs">
+      <Link>Common\System\IO\StringParser.cs</Link>
+    </Compile>
+    <Compile Include="$(CommonPath)\Interop\Linux\Interop.LinuxNetDeviceFlags.cs">
+      <Link>Interop\Linux\Interop.LinuxNetDeviceFlags.cs</Link>
+    </Compile>
+    <Compile Include="$(CommonPath)\Interop\Linux\Interop.LinuxTcpState.cs">
+      <Link>Interop\Linux\Interop.LinuxTcpState.cs</Link>
+    </Compile>
+    <Compile Include="$(CommonPath)\Interop\Linux\System.Native\Interop.NetworkChange.cs">
+      <Link>Interop\Linux\System.Native\Interop.NetworkChange.cs</Link>
+    </Compile>
   </ItemGroup>
-  
+
+  <!-- OSX -->
+  <ItemGroup Condition=" '$(TargetsOsx)' == 'true'">
+    <Compile Include="System\Net\NetworkInformation\IPGlobalPropertiesPal.Osx.cs" />
+    <Compile Include="System\Net\NetworkInformation\NetworkAddressChange.OSX.cs" />
+    <Compile Include="System\Net\NetworkInformation\NetworkInterfacePal.Osx.cs" />
+    <Compile Include="System\Net\NetworkInformation\OsxIcmpV4Statistics.cs" />
+    <Compile Include="System\Net\NetworkInformation\OsxIcmpV6Statistics.cs" />
+    <Compile Include="System\Net\NetworkInformation\OsxIPGlobalProperties.cs" />
+    <Compile Include="System\Net\NetworkInformation\OsxIPv4GlobalStatistics.cs" />
+    <Compile Include="System\Net\NetworkInformation\OsxIpInterfaceStatistics.cs" />
+    <Compile Include="System\Net\NetworkInformation\OsxIPv4InterfaceProperties.cs" />
+    <Compile Include="System\Net\NetworkInformation\OsxIPv6InterfaceProperties.cs" />
+    <Compile Include="System\Net\NetworkInformation\OsxIpInterfaceProperties.cs" />
+    <Compile Include="System\Net\NetworkInformation\OsxNetworkInterface.cs" />
+    <Compile Include="System\Net\NetworkInformation\OsxTcpStatistics.cs" />
+    <Compile Include="System\Net\NetworkInformation\OsxUdpStatistics.cs" />
+    <!-- OSX Common -->
+    <Compile Include="$(CommonPath)\Interop\OSX\System.Native\Interop.ProtocolStatistics.cs">
+      <Link>Interop\OSX\System.Native\Interop.ProtocolStatistics.cs</Link>
+    </Compile>
+    <Compile Include="$(CommonPath)\Interop\OSX\System.Native\Interop.TcpConnectionInfo.cs">
+      <Link>Interop\OSX\System.Native\Interop.TcpConnectionInfo.cs</Link>
+    </Compile>
+  </ItemGroup>
+
   <ItemGroup>
     <None Include="project.json" />
   </ItemGroup>
+
   <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory), dir.targets))\dir.targets" />
-</Project>
+</Project>
```

Comment:
NIT: Missing NL (although I noticed that VS doesn't like adding it so maybe we leave it as is?).